src/RacingGameManager.js

```diff
@@ -0,0 +1,58 @@
+import { Console } from '@woowacourse/mission-utils';
+import Car from './Car.js';
+import { CONSOLE_MESSAGE } from './constant.js';
+import { printEmptyString } from './util.js';
+
+class RacingGameManager {
+  #cars;
+  #tryCount;
+
+  constructor(carNames, tryCount) {
+    this.#cars = carNames.map((carName) => new Car(carName));
+    this.#tryCount = tryCount;
+  }
+
+  playGame() {
+    this.#startGame();
+
+    const winners = this.#selectWinner();
+    this.#printWinner(winners);
+  }
+
+  #startGame() {
+    Console.print(CONSOLE_MESSAGE.GAME_START_MESSAGE);
+    for (let i = 0; i < this.#tryCount; i++) {
+      this.#printTryResult();
+      printEmptyString();
+    }
+  }
+
+  #printTryResult() {
+    this.#cars.forEach((car) => {
+      car.move();
+      car.printCarPosition();
+    });
+  }
+
+  #selectWinner() {
+    const maxCarPosition = Math.max(
+      ...this.#cars.map((car) => car.getPosition()),
+    );
```

Comment:
```suggestion
    const carPositions = [...this.#cars.map((car) => car.getPosition())]
    const maxCarPosition = Math.max(carPositions);
```

이렇게 수정하면 의미를 더 명확하게 설명해 줄 수 있으면서, 가독성이 올라가 것 같습니다! 혹시 어떻게 생각하시나요?